vterm.el

```diff
@@ -298,7 +298,10 @@ If nil, never delay")
   (if vterm-copy-mode
       (progn                            ;enable vterm-copy-mode
         (use-local-map nil)
-        (vterm-send-stop))
+        (vterm-send-stop)
+        (set (make-local-variable 'vterm-copy-saved-point) (point)))
```

Comment:
I think you should define a local variable  like `vterm--process` 
and I suggest you rename  `vterm-copy-saved-point` to `vterm--copy-saved-point`

Reply:
Thank, I will do that shortly.

Reply:
All updated with your suggestions, thanks!